### DISCUSSION TITLE
Roms combining into files that I can't modify

### DISCUSSION THREAD

**Discussion by spiegel22** [OnionUI/Onion#1726] [Q&A] [answered]
Hello!  I'm new to the Miyoo Mini+ and Onion, but have been using both for about two weeks now.  I self-installed the OS on a new Sandisk card, updated the system, etc and everything was going fine.  My games had been working fine, until I tried to launch what appears to be a badly formatted custom rom and got stuck in the Death Loop of the Mini rebooting into the last save state.  I broke out of it by following the guides I found online, but was left with a strange file in the ROMs folder, as attached.

For a bit none of my SNES games worked anymore, but after a few reboots they started working again.  Now I don't see any of my game files or Imgs folder, but instead this strange file.  I tried re-adding the games from my backups which told me that the files were already present -- again, I cannot see them at all.  I overwrote those files and now I have TWO of these mystery files....

Has anyone seen this before or have any guidance?
![image](https://github.com/user-attachments/assets/b8abdfd3-2793-43b3-97ca-81f3726b3444)
![image](https://github.com/user-attachments/assets/f029baf9-9985-4a6a-a52b-54c132b64a5c)
![image](https://github.com/user-attachments/assets/598dd527-7f7f-46af-b1fb-1dd37ebc804a)

I can neither move nor delete this thing by conventional means and am hoping not to have to reformat to get rid of this.

**Comment by thomasklemenc** [comment, marked as answer]
This looks like a corrupt/dead SD card

you can try  `chkdsk E: /f` (replace E: with your SD card letter)

**Reply by Aemiii91** [reply]
First sentence is the answer. You'll need to reformat.

**Comment by spiegel22** [comment]
Hoping it's not the SD card, although it is brand new so I suppose I could get a refund or replacement if the issues continue.

Lots of weirdness in there, but the keys were in the SFC folder, which it renamed and reallocated into those DEMON'~1.zi0 files, which I was then able to delete.  All of this started when I got into the reboot loop, so hopefully this will be the end of it!  Thanks for the tip!


-------------
The type of the file system is FAT32.
Volume ONION OS created 12/11/2024 10:36 PM
Volume Serial Number is 1E0C-2D52
Windows is verifying files and folders...
\Roms\SFC\USBC`j↑I  first allocation unit is not valid. The entry will be truncated.
More than one DEMON'~1.ZIP entry in folder \Roms\SFC.
Renamed to DEMON'~1.ZI0.
More than one DEMON'~1.ZIP entry in folder \Roms\SFC.
Renamed to DEMON'~1.ZI0.
\Roms\SFC\USBC► }<  first allocation unit is not valid. The entry will be truncated.
Removing trailing folder entries from \Roms\SFC
The \Roms\GBA\Imgs\<Game 1>.png entry contains a nonvalid link.
The size of the \Roms\GBA\Imgs\<Game 1>.png entry is not valid.
The \Roms\GBA\Imgs\<Game 2>.png entry contains a nonvalid link.
The size of the \Roms\GBA\Imgs\<Game 2>.png entry is not valid.
\Roms\GBA\Imgs\<Game 3>.png is cross-linked on allocation unit 200717.
Cross link resolved by copying.
\Saves\CurrentProfile\states\Gearboy\<Game 4>.state is cross-linked on allocation unit 200723.
Cross link resolved by copying.
\Saves\CurrentProfile\states\Gearboy\Legend of the River King GB (Stream).auto is cross-linked on allocation unit 200724.
Cross link resolved by copying.
\Saves\CurrentProfile\states\Gearboy\Legend of the River King GB (Stream).state is cross-linked on allocation unit 200725.
Cross link resolved by copying.
File and folder verification is complete.
Bad links in lost chain at cluster 200705 corrected.
Lost chain cross-linked at cluster 200706.  Orphan truncated.
Bad links in lost chain at cluster 200707 corrected.
Bad links in lost chain at cluster 200708 corrected.
Bad links in lost chain at cluster 200709 corrected.
Bad links in lost chain at cluster 3875072 corrected.
Bad links in lost chain at cluster 3875073 corrected.
Lost chain cross-linked at cluster 3875074.  Orphan truncated.
Bad links in lost chain at cluster 3875075 corrected.
Bad links in lost chain at cluster 3875076 corrected.
Bad links in lost chain at cluster 3875077 corrected.
Bad links in lost chain at cluster 3875080 corrected.
Bad links in lost chain at cluster 3875081 corrected.
Bad links in lost chain at cluster 3875082 corrected.
Bad links in lost chain at cluster 3875083 corrected.
Bad links in lost chain at cluster 3875084 corrected.
Bad links in lost chain at cluster 3875085 corrected.
Bad links in lost chain at cluster 3875086 corrected.
Bad links in lost chain at cluster 3875087 corrected.
Bad links in lost chain at cluster 3875088 corrected.
Bad links in lost chain at cluster 3875089 corrected.
Bad links in lost chain at cluster 3875090 corrected.
Bad links in lost chain at cluster 3875091 corrected.
Bad links in lost chain at cluster 3875092 corrected.
Bad links in lost chain at cluster 3875093 corrected.
Bad links in lost chain at cluster 3875094 corrected.
Bad links in lost chain at cluster 3875096 corrected.
Bad links in lost chain at cluster 3875097 corrected.
Bad links in lost chain at cluster 3875098 corrected.
Bad links in lost chain at cluster 3875099 corrected.
Bad links in lost chain at cluster 3875100 corrected.
Bad links in lost chain at cluster 3875101 corrected.
Lost chain cross-linked at cluster 3875102.  Orphan truncated.
Bad links in lost chain at cluster 3875103 corrected.
Bad links in lost chain at cluster 3875104 corrected.
Bad links in lost chain at cluster 3875105 corrected.
Bad links in lost chain at cluster 3875106 corrected.
Bad links in lost chain at cluster 3875107 corrected.
Bad links in lost chain at cluster 3875108 corrected.
Bad links in lost chain at cluster 3875109 corrected.
Bad links in lost chain at cluster 3875110 corrected.
Bad links in lost chain at cluster 3875111 corrected.
Lost chain cross-linked at cluster 3875112.  Orphan truncated.
Bad links in lost chain at cluster 3875113 corrected.
Bad links in lost chain at cluster 3875115 corrected.
Bad links in lost chain at cluster 3875118 corrected.
Bad links in lost chain at cluster 3875120 corrected.
Lost chain cross-linked at cluster 3875121.  Orphan truncated.
Bad links in lost chain at cluster 3875122 corrected.
Bad links in lost chain at cluster 3875123 corrected.
Bad links in lost chain at cluster 3875124 corrected.
Bad links in lost chain at cluster 3875125 corrected.
Lost chain cross-linked at cluster 3875126.  Orphan truncated.
Bad links in lost chain at cluster 3875127 corrected.
Bad links in lost chain at cluster 3875128 corrected.
Bad links in lost chain at cluster 3875129 corrected.
Bad links in lost chain at cluster 3875130 corrected.
Bad links in lost chain at cluster 3875131 corrected.
Bad links in lost chain at cluster 3875132 corrected.
Bad links in lost chain at cluster 3875133 corrected.
Lost chain cross-linked at cluster 3875134.  Orphan truncated.
Bad links in lost chain at cluster 3875135 corrected.
Bad links in lost chain at cluster 3875136 corrected.
Bad links in lost chain at cluster 3875137 corrected.
Bad links in lost chain at cluster 3875138 corrected.
Bad links in lost chain at cluster 3875139 corrected.
Bad links in lost chain at cluster 3875140 corrected.
Bad links in lost chain at cluster 3875141 corrected.
Lost chain cross-linked at cluster 3875142.  Orphan truncated.
Bad links in lost chain at cluster 3875143 corrected.
Bad links in lost chain at cluster 3875144 corrected.
Bad links in lost chain at cluster 3875145 corrected.
Bad links in lost chain at cluster 3875146 corrected.
Bad links in lost chain at cluster 3875147 corrected.
Bad links in lost chain at cluster 3875148 corrected.
Bad links in lost chain at cluster 3875149 corrected.
Bad links in lost chain at cluster 3875150 corrected.
Bad links in lost chain at cluster 3875151 corrected.
Bad links in lost chain at cluster 3875152 corrected.
Bad links in lost chain at cluster 3875153 corrected.
Bad links in lost chain at cluster 3875154 corrected.
Bad links in lost chain at cluster 3875155 corrected.
Bad links in lost chain at cluster 3875156 corrected.
Bad links in lost chain at cluster 3875157 corrected.
Bad links in lost chain at cluster 3875160 corrected.
Lost chain cross-linked at cluster 3875161.  Orphan truncated.
Bad links in lost chain at cluster 3875162 corrected.
Bad links in lost chain at cluster 3875163 corrected.
Bad links in lost chain at cluster 3875164 corrected.
Bad links in lost chain at cluster 3875165 corrected.
Lost chain cross-linked at cluster 3875166.  Orphan truncated.
Bad links in lost chain at cluster 3875167 corrected.
Bad links in lost chain at cluster 3875168 corrected.
Bad links in lost chain at cluster 3875169 corrected.
Bad links in lost chain at cluster 3875170 corrected.
Bad links in lost chain at cluster 3875171 corrected.
Bad links in lost chain at cluster 3875172 corrected.
Bad links in lost chain at cluster 3875173 corrected.
Lost chain cross-linked at cluster 3875174.  Orphan truncated.
Bad links in lost chain at cluster 3875175 corrected.
Bad links in lost chain at cluster 3875176 corrected.
Bad links in lost chain at cluster 3875177 corrected.
Bad links in lost chain at cluster 3875178 corrected.
Bad links in lost chain at cluster 3875179 corrected.
Bad links in lost chain at cluster 3875180 corrected.
Bad links in lost chain at cluster 3875181 corrected.
Lost chain cross-linked at cluster 3875182.  Orphan truncated.
Bad links in lost chain at cluster 3875183 corrected.
Bad links in lost chain at cluster 3875184 corrected.
Bad links in lost chain at cluster 3875185 corrected.
Bad links in lost chain at cluster 3875186 corrected.
Bad links in lost chain at cluster 3875187 corrected.
Bad links in lost chain at cluster 3875188 corrected.
Bad links in lost chain at cluster 3875189 corrected.
Bad links in lost chain at cluster 3875190 corrected.
Bad links in lost chain at cluster 3875191 corrected.
Bad links in lost chain at cluster 3875192 corrected.
Bad links in lost chain at cluster 3875193 corrected.
Bad links in lost chain at cluster 3875194 corrected.
Bad links in lost chain at cluster 3875195 corrected.
Lost chain cross-linked at cluster 3875196.  Orphan truncated.
Bad links in lost chain at cluster 3875198 corrected.
Convert lost chains to files (Y/N)? y
19616 KB in 178 recovered files.

Windows has made corrections to the file system.
No further action is required.
  124,948,448 KB total disk space.
           32 KB in 1 hidden files.
       41,952 KB in 1,311 folders.
    7,514,432 KB in 5,723 files.
  117,392,000 KB are available.

       32,768 bytes in each allocation unit.
    3,904,639 total allocation units on disk.
    3,668,500 allocation units available on disk.
